tests/src/coffee_integration_tests.rs

```diff
@@ -289,3 +289,78 @@ pub async fn test_errors_and_show() {
 
     cln.stop().await.unwrap();
 }
+
+#[tokio::test]
+#[ntest::timeout(560000)]
+pub async fn install_plugin_in_two_networks() -> anyhow::Result<()> {
+    init();
+    // initialize a lightning node in regtest network
+    let mut cln = Node::tmp("regtest".to_string()).await.unwrap();
+    let lightning_dir = cln.rpc().getinfo().unwrap().ligthning_dir;
+    let lightning_regtest_dir = lightning_dir.strip_suffix("/regtest").unwrap();
+    log::info!("lightning path for regtest network: {lightning_dir}");
+
+    let dir = Arc::new(tempfile::tempdir()?);
+    let args = CoffeeTestingArgs {
+        conf: None,
+        data_dir: dir.path().clone().to_str().unwrap().to_owned(),
+        network: "regtest".to_string(),
+    };
+    let mut manager = CoffeeTesting::tmp_with_args(&args, dir.clone()).await?;
+    manager
+        .coffee()
+        .setup(&lightning_regtest_dir)
+        .await
+        .unwrap();
+    // Add lightningd remote repository
+    manager
+        .coffee()
+        .add_remote("lightningd", "https://github.com/lightningd/plugins.git")
+        .await
+        .unwrap();
+    // Install summary plugin
+    // This should install summary plugin for regtest network
+    manager
+        .coffee()
+        .install("summary", true, true)
+        .await
+        .unwrap();
+    // Ensure that summary is installed for regtest network
+    cln.rpc()
+        .call::<serde_json::Value, serde_json::Value>("summary", json!({}))
+        .unwrap();
+    cln.stop().await.unwrap();
+
+    // dropping the first coffee instance, but without delete the dir
+    drop(manager);
+    // initialize a lightning node in testnet network
+    let mut cln = Node::tmp("testnet".to_string()).await.unwrap();
+    let lightning_dir = cln.rpc().getinfo().unwrap().ligthning_dir;
+    let lightning_testnet_dir = lightning_dir.strip_suffix("/testnet").unwrap();
+    log::info!("lightning path for testnet network: {lightning_dir}");
+    let new_args = CoffeeTestingArgs {
+        conf: None,
+        data_dir: dir.path().clone().to_string_lossy().to_string(),
+        network: "testnet".to_string(),
+    };
```

Comment:
we can assert on the dir? and make sure that you are running coffee on the same directory.